server/routes/reporting/standard-report/definitions.js

```diff
@@ -367,6 +368,47 @@ const { dailyUtilisationDAO, dailyUtilisationJurorsDAO } = require('../../../obj
           totals: true,
         },
       },
+      'prepare-monthly-utilisation': {
+        title: 'Monthly wastage and utilisation report',
+        headings: [
+          'dateFrom',
+          'reportDate',
+          'dateTo',
+          'reportTime',
+          '',
+          'courtName',
+        ],
+        bespokeReport: {
+          dao: async(req) => await generateMonthlyUtilisationDAO.get(
```

Comment:
update to `dao: (req) => generateMonthlyUtilisationDAO.get(`

Reply:
updated.